development-docs/systemtests/io.strimzi.systemtest.operators.user.UserST.md

```diff
@@ -0,0 +1,136 @@
+# UserST
+
+**Description:** Test suite for various Kafka User operations and configurations.
+
+**Before tests execution steps:**
```

Comment:
```suggestion
**Before test execution steps:**
```

Reply:
This is currently hardcoded in our documentation tool. However, I don’t understand why `test` is used instead of `tests`, since the description applies to all tests included in the test execution.

Reply:
I guess it's just typo.